packages/fx-core/resource/package.nls.json

```diff
@@ -852,6 +852,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. The length of client secret should be in this range: >=10 and <=128",
   "driver.apiKey.error.domainInvalid": "Domain is invalid. Domain for API key should follow: 1. Max %d domain(s) per API key. 2. Use comma to separate domains",
```

Comment:
Proposed: Invalid domain. Please follow these rules: 1. Max %d domain(s) per API key. 2. Use comma to separate domains.

Reply:
Updated, thanks!